lib/desktop/session.rb

```diff
@@ -270,13 +270,25 @@ def start_grabber
     def start_vnc_server(geometry: Config.geometry)
       IO.popen(
         {}.tap do |h|
-          h['flight_DESKTOP_root'] = Config.root
           h['flight_DESKTOP_type_root'] = type.dir
-          h['flight_DESKTOP_bg_image'] = File.expand_path(
-            Config.bg_image,
-            Config.root
-          )
+          h['flight_DESKTOP_bg_image'] = Config.bg_image
           h['flight_DESKTOP_geometry'] = geometry
+
+          # DEPREACTED: flight_DESKTOP_root does not function correctly
+          # due to the desktop-types being extracted. Originally it was
+          # used to construct paths like:
+          #
+          # etc/gnome/session.sh:flight_DESKTOP_type_root="${flight_DESKTOP_type_root:-${flight_DESKTOP_root}/etc/types/gnome}"
+          #
+          # However flight-desktop-type are no longer stored under 'etc',
+          # so this paths isn't correct. The builder also changes the path
+          # to be within 'usr/lib'. AFAICS, this fallback is never correct.
+          #
+          # Instead flight_DESKTOP_root should be removed entirely from
+          # flight-desktop{-type} in favour of setting
+          # `flight_DESKTOP_type_root and flight_DESKTOP_bg_image
+          # directly; as done above.
+          h['flight_DESKTOP_root'] = Config.root
```

Comment:
I'm unclear as to why this is deprecated instead of being removed.  Are the desktop types still making use of this?  Or have they all been updated?

Reply:
The `etc/gnome/session.sh` scripts still use it as a "default" yes. However if the default was every triggered by the production build, it would be wrong.

I was hesitant to remove it because:

* It would *technically* be breaking,
* It *maybe* in use elsewhere, and
* It is somewhat unrelated to to this PR.

Reply:
tl;dr

It was already `deprecated` all I did was document it.